src/omero_zarr/cli.py

```diff
@@ -294,7 +308,21 @@ def _bf_export(self, abs_path: Path, args: argparse.Namespace) -> None:
         if stderr:
             self.ctx.err(stderr.decode("utf-8"))
         if process.returncode == 0:
-            self.ctx.out(f"Image exported to {target.resolve()}")
+            image_source = target / "0"
+            image_source.rename(image_target)
+            target.rmdir()
+            self.ctx.out(f"Image exported to {image_target.resolve()}")
+
+        # Add OMERO metadata
+        store = FSStore(
+            str(image_target.resolve()),
+            auto_mkdir=False,
+            normalize_keys=False,
+            mode="w",
+        )
+        root = open_group(store)
+        add_omero_metadata(root, image)
+        add_toplevel_metadata(root)
```

Comment:
this just sets `_creator`. Is it almost not more appropriately bf2raw, or a mix?

Reply:
Definitely a mix. There is some Bio-Formats metadata under `multiscales/metadata` but more could be captured.
Happy to start capturing that but I am a bit worried you'll ask me to solve https://github.com/ome/omero-cli-zarr/issues/48 ;)

Reply:
If you _want_ to, ... :wink: